cmd/thanos/query.go

```diff
@@ -200,7 +204,8 @@ func runQuery(
 	webRoutePrefix string,
 	webExternalPrefix string,
 	webPrefixHeaderName string,
-	maxConcurrentQueries int,
+	_ int, // maxConcurrentQueries.
```

Comment:
@bwplotka I have realized this isn't used at all. Is it intentional? Maybe reserved for future use?

Reply:
no idea what's this